README.md

```diff
@@ -42,15 +42,16 @@ Binrc supports settings binary version numbers as environment variables. It will
 in the environment to configure the right version.
 
 ```
-HUGO_VERSION=0.19 binrc install spf13/hugo
+HUGO_VERSION=0.94.2 binrc install gohugoio/hugo
```

Comment:
Well… this version is already a year old. Would make sense to make it more current before merging.

Related: _(question is… whether there should not be a "latest" option?)_
- #39